gui/mainwindow.cpp

```diff
@@ -79,11 +79,20 @@ MainWindow::MainWindow(TranslationHandler* th, QSettings* settings) :
     mLineEditFilter = new QLineEdit(mUI.mToolBarFilter);
     mLineEditFilter->setPlaceholderText(tr("Quick Filter:"));
     mUI.mToolBarFilter->addWidget(mLineEditFilter);
-    connect(mLineEditFilter, SIGNAL(textChanged(const QString&)), mFilterTimer, SLOT(start()));
+#ifndef QOVERLOAD_FALLBACK
+    connect(mLineEditFilter, &QLineEdit::textChanged, mFilterTimer, QOverload<>::of(&QTimer::start));
+#else
+    connect(mLineEditFilter, &QLineEdit::textChanged, mFilterTimer, static_cast<void (QTimer::*)(void)>(&QTimer::start));
```

Comment:
I am not sure if this is really safe for all possible calling conventions. For instance if the return address and arguments would be pushed on the stack I envision that the stack is not read/written properly if arguments are just skipped.

I am guessing that a lambda function can be used to ignore the argument safely.